Examples/SubscriberExample/Sources/Screens/Map/MapViewController.swift

```diff
@@ -136,6 +134,27 @@ class MapViewController: UIViewController {
             mapView.addAnnotation(annotation)
         }
     }
+    
+    private func updateHorizontalAccuracyAnnotation(position: Position) {
+        let coordinate = CLLocationCoordinate2D(latitude: position.latitude, longitude: position.longitude)
+        let region = MKCoordinateRegion(center: coordinate, latitudinalMeters: position.accuracy, longitudinalMeters: position.accuracy)
+        let rect = mapView.convert(region, toRectTo: mapView)
+
+        if let annotation = mapView.annotations.first(where: { $0 is HorizontalAccuracyAnnotation }) as? HorizontalAccuracyAnnotation  {
+            annotation.accuracy = position.accuracy
+            
+            if let view = mapView.view(for: annotation) as? HorizontalAccuracyAnnotationView {
+                view.accuracy = Double(rect.size.width)
+            }
+            
+            annotation.coordinate = coordinate
+        } else {
+            let annotation = HorizontalAccuracyAnnotation()
+            annotation.coordinate = coordinate
+            annotation.accuracy = position.accuracy
```

Comment:
Both in the `if` and `else` you are setting the `annotation.coordinate` and `annotation.accuracy`, maybe it would be possible to extract it out of this `if/esle` to avoid code duplication? :thinking:  Something like: 
(Warning: pseudo-Swift below :sweat_smile: )
```
if let annotation = mapView.annotations.first(where: { $0 is HorizontalAccuracyAnnotation }) as? HorizontalAccuracyAnnotation  {
    if let view = mapView.view(for: annotation) as? HorizontalAccuracyAnnotationView {
        view.accuracy = Double(rect.size.width)
    }
    } else {
        let annotation = HorizontalAccuracyAnnotation()
        mapView.addAnnotation(annotation)
    }
annotation.coordinate = coordinate
annotation.accuracy = position.accuracy
```

Reply:
I've extracted annotation creation to the generic function. Changed here c21e66965c43f6eb9a53d314a4253544a03db53c